lms/djangoapps/grades/rest_api/v1/gradebook_views.py

```diff
@@ -551,12 +552,29 @@ def get(self, request, course_key):
                     q_objects.append(Q(user__in=[]))
             if request.GET.get('enrollment_mode'):
                 q_objects.append(Q(mode=request.GET.get('enrollment_mode')))
+            if request.GET.get('assignment_grade_max') and request.GET.get('assignment'):
+                # TODO: support both max and min, consolidate shared logic between them to make query efficient
+                # TODO: I probably want lte and gte, here and elsewhere
+                from pudb import set_trace; set_trace()
```

Comment:
😱

Reply:
I put wip in like five different places can you calm down